tests/models/resource_test.py

```diff
@@ -486,6 +486,13 @@ def test_recalculate_descriptors_prefetch_related_objects(self):
                 ]
                 self.assertEqual(len(tile_selects), 1)
 
+                non_guardian_user_selects = [
+                    q
+                    for q in queries
+                    if q["sql"].endswith('FROM "auth_user"') and "guardian" not in q
+                ]
+                self.assertEqual(len(non_guardian_user_selects), 1)
```

Comment:
This is 1 of 2 new assertions -- the rest is cleanup (more obvious if you review by commit)